docs/learn/learn.md

```diff
@@ -0,0 +1 @@
+package learn
```

Comment:
Havent figured out a way of adding a `docsId` to the pages/learn yet. Tried inserting some react but this is a bit of a hacky workaround for now to ensure the versioning works.